### DISCUSSION TITLE
[destination-google-sheets] Airbyte updates is messing up with my reference cells in formulas

### DISCUSSION THREAD

**Discussion by faprusso** [airbytehq/airbyte#35671] [Connector Questions] [answered]
Hey guys, 

new to Airbyte, still getting used. I have made a conection between my Source (API built connector) and my Google Sheets. The data comes from airbyte into my spreadsheet and i have a dashboard sheet to work with the data. But every update my formulas references are being modified. What is going on?

Example: in my Dashboard sheet i make a reference to cell B2 in the spreadsheet that receives the data from Airbyte. But after uptade i check the formula and now the reference has changed to b746. 

Cheers!

**Comment by rowanmoul** [comment]
Try locking your cell references. I suspect Airbyte is using an _insert row_ operation to add rows to your reference sheet, which will adjust formulas to continue pointing to the same cell they were before. For example if you are referring to cell B2 and you insert a row before row 2, then the inserted row will become row 2, and the row that used to be row 2 will now be row 3.
I found this post on how to lock your references:
https://support.google.com/docs/thread/4440337/how-do-you-lock-a-reference-cell-in-a-formula-when-i-autofill-and-don-t-want-reference-cell-2-move?hl=en

**Reply by faprusso** [reply, marked as answer]
Thanks for your reply. Airbyte does insert row in every update. The way i found to fix that in google sheets is using INDIRECT function, so it fixes the cell reference. 

Cheers
